Apps/Admin/Server/Models/AdminUserProfileView.cs

```diff
@@ -60,6 +60,6 @@ public class AdminUserProfileView
         /// <summary>
         /// Gets or sets the user's last login date time from DB AdminUserProfile.
         /// </summary>
-        public DateTime? LastLoginDateTime { get; set; }
+        public DateTime LastLoginDateTime { get; set; }
```

Comment:
Why is this file being changed? Is the Support Page using this view? I believe it’s being used by inactive users report?